patch-diff-report-tool/src/main/resources/maven-base.css

```diff
@@ -26,6 +26,9 @@ div.clear{
 div.clear hr{
   display: none;
 }
+:target {
```

Comment:
This should be `a:target` to show that you want to only highlight links.

Reply:
done, tested, works good